spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'User Show Page', type: :feature do
+  before(:each) do
+    @user = User.create!(name: 'Example User', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', posts_counter: 3,
+                         bio: 'I am a user')
```

Comment:
- [ ] Good job so far team 🥇, however, Remember to add these three tests, as we can see to the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20user%27s%20first,me%20to%20that%20post%27s%20show%20page.). Kindly add them 👍 

````md
- I can see the user's first 3 posts.
- I can see a button that lets me view all of a user's posts.
- When I click a user's post, it redirects me to that post's show page.
````